settings_public_or_non_public.py-dist

```diff
@@ -0,0 +1,10 @@
+# Django settings for public instance of the WDE desktop_disk_encrypter project.
+
+# Allow users to access the administrative interface
+ALLOW_ADMIN = True
+
+# This is used to provide cryptographic signing, and should be set to a unique, unpredictable value.
+SECRET_KEY = ''
+
+# Users must be a member of this group to access the admin site
+OIDC_DESKTOP_CLAIM_GROUP = ''
```

Comment:
I would add some safety valve code here:

```python
if (ALLOW_ADMIN == True and (SECRET_KEY == '' or OIDC_DESKTOP_CLAIM_GROUP == ''):
  raise AwoogaException
```

Reply:
Django raises
```
ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```

If `SECRET_KEY` is empty.

I've added #9 to address an empty `OIDC_DESKTOP_CLAIM_GROUP`

It would still be possible for someone to configure `ALLOW_ADMIN == True` and leave `OIDC_DESKTOP_CLAIM_GROUP` empty, it would just mean nobody could log in